lib/kafka/cluster.rb

```diff
@@ -310,8 +293,7 @@ def resolve_offsets(topic, partitions, offset)
             topic => broker_partitions.map {|partition|
               {
                 partition: partition,
-                time: offset,
-                max_offsets: 1,
+                time: offset
```

Comment:
@dasch I bumped the version of ListOffset API so that it supports the Isolation Level. The new version of this API drops the support of `max_num_offsets`. It returns the latest offset, instead of an array.

Reply:
:+1: